src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -363,7 +363,7 @@ public override bool AreLocalsZeroed
             get
             {
                 var data = this.GetDecodedWellKnownAttributeData();
-                return data != null ? !data.HasSkipLocalsInitAttribute : true;
+                return data != null ? !data.HasSkipLocalsInitAttribute : ContainingType.AreLocalsZeroed;
```

Comment:
Is this tested anywhere? Is it currently possible for `ContainingType.AreLocalsZeroed` to return `false` for a `SourceMemberMethodSymbol`?

Reply:
It is currently not possible for the ContainingType to return false, but it will be once we enable applying SkipLocalsInitAttribute directly on a class. The best we can do to test it for now is already done in `SkipLocalsInitAttributeOnMethod`: we make sure that the locals are being initialized when the attribute is not applied (AreLocalsZeroed returns true).